Documentation/ceph-monitoring.md

```diff
@@ -7,66 +7,76 @@ indent: true
 # Monitoring
 
 Each Rook Ceph cluster has some built in metrics collectors/exporters for monitoring with [Prometheus](https://prometheus.io/).
+
 If you do not have Prometheus running, follow the steps below to enable monitoring of Rook. If your cluster already
 contains a Prometheus instance, it will automatically discover Rooks scrape endpoint using the standard
 `prometheus.io/scrape` and `prometheus.io/port` annotations.
 
+> **NOTE** This assumes that the Prometheus instances is searching all your Kubernetes namespaces for Pods with these annotations.
 
 ## Prometheus Operator
 
 First the Prometheus operator needs to be started in the cluster so it can watch for our requests to start monitoring Rook and respond by deploying the correct Prometheus pods and configuration.
 A full explanation can be found in the [Prometheus operator repository on GitHub](https://github.com/coreos/prometheus-operator), but the quick instructions can be found here:
-```bash
+
+```console
 kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.26.0/bundle.yaml
 ```
+
 This will start the Prometheus operator, but before moving on, wait until the operator is in the `Running` state:
-```bash
+
+```console
 kubectl get pod
 ```
-Once the Prometheus operator is in the `Running` state, proceed to the next section.
+
+Once the Prometheus operator is in the `Running` state, proceed to the next section to create a Prometheus instance.
 
 ## Prometheus Instances
 
 With the Prometheus operator running, we can create a service monitor that will watch the Rook cluster and collect metrics regularly.
 From the root of your locally cloned Rook repo, go the monitoring directory:
-```bash
+
+```console
 cd cluster/examples/kubernetes/ceph/monitoring
 ```
 
 Create the service monitor as well as the Prometheus server pod and service:
-```bash
+
+```console
 kubectl create -f service-monitor.yaml
 kubectl create -f prometheus.yaml
 kubectl create -f prometheus-service.yaml
 ```
 
 Ensure that the Prometheus server pod gets created and advances to the `Running` state before moving on:
-```bash
+
+```console
 kubectl -n rook-ceph get pod prometheus-rook-prometheus-0
 ```
 
+> **NOTE** It is not a good idea to use storage from the Ceph cluster that is monitored for Prometheus.
+> Because if the Ceph cluster would fail, your Prometheus can become unresponsible because of that and thus not alert you.
```

Comment:
how about this?
"It is not recommended to consume storage from the Ceph cluster for Prometheus.
If the Ceph cluster fails, Prometheus would become unresponsive and thus not alert you of the failure."